docs/test.ts

```diff
@@ -371,7 +372,7 @@ tracer.trace('test', { service: 'foo', resource: 'bar', type: 'baz' }, () => {})
 tracer.trace('test', { measured: true }, () => {})
 tracer.trace('test', (span: Span) => {})
 tracer.trace('test', (span: Span, fn: () => void) => {})
-tracer.trace('test', (span: Span, fn: (err: Error) => string) => {})
+tracer.trace('test', (span: Span, fn: (err: Error) => void) => {})
```

Comment:
that means the callback is supposed to return void right ?

Reply:
Not an _explicit_ `return void`, if that's what you're asking. Just the absence of a return. It being a string before was _probably_ an error before but I don't know the history. Either way, it's been no-return for as long as I'm aware of, so the types should match.

Reply:
What about making it `any` ? sometimes people write stuff like:
```javascript
if (err) {
  return something(err)
}
```
and something() might return a non null

Reply:
Sure, I made it `any`.

Reply:
I still see `void` here, did you push your commit ?

Reply:
I realized it would actually _always_ be `void`. The `done` function is coming from _us_ and it never has a return. 😅